rules/windows/process_creation/proc_creation_win_unusual_child_process_of_dns_exe.yml

```diff
@@ -0,0 +1,23 @@
+title: Unusual Child Porcess of dns.exe
+id: a4e3d776-f12e-42c2-8510-9e6ed1f43ec3
+status: experimental
+description: Detects an unexpected process spawning from dns.exe which may indicate activity related to remote code execution or other forms of exploitation as seen in CVE-2020-1350 (SigRed)
+references:
+    - https://www.elastic.co/guide/en/security/current/unusual-child-process-of-dns.exe.html
+author: Tim Rauch
+date: 2022/09/27
+logsource:
+    category: process_creation
+    product: windows
+tags:
+    - attack.initial_access
+    - attack.t1133
+detection:
+    selection:
+        ParentImage|endswith: '\dns.exe'
+    selection_not:
```

Comment:
Using `filter` instead of "selection_not" is a best practice for the comprehension